CHANGELOG.md

```diff
@@ -2,13 +2,19 @@
 
 ## v2.9.0
 
+### Routing
+
+* `RouteController.prefersOnlineRoute` enables autmoatical switching from `onboard` generated to `online` generated route if it's geometry is the same. Use `NavigationViewControllerDelegate.navigationViewController(_:didSwitchToCoincideOnlineRoute:)`, `NavigationServiceDelegate.navigationService(_:didSwitchToCoincideOnlineRoute:)`, `RouterDelegate.router(_:didSwitchToCoincideOnlineRoute:)` or `.routeControllerDidSwitchToCoincideOnlineRoute` notification to track such events. ([#4127](https://github.com/mapbox/mapbox-navigation-ios/pull/4127))
+* Added `NavigationViewController.init(for:routeIndex:navigationOptions:)` to start navigation using map matching response. ([#4127](https://github.com/mapbox/mapbox-navigation-ios/pull/4127))
```

Comment:
No. Inits with `IndexedRouteResponse` do not take route index and options since it is already built in. There are similar initializers for `RouteResponse` and `MapMatchingResponse` but the former one is deprecated and is not recommended to use.